etc/metrics/org.osg.general.perfsonar-simple.conf

```diff
@@ -4,17 +4,22 @@ probe-interval = 60
 job-timeout   = 1500
 no-ping = True
 [org.osg.general.perfsonar-simple args]
-username = perfsonar
-key = ab132837b1f49139e7d1f1f635956ee15c37068e
+username = perfsonar3
+#This is the good key
+key = a4dfdf481a73f37b9dad39528c6150f2944e0934
+# this is the wrong key
+#key = ab132837b1f49139e7d1f1f635956ee15c37068
 goc = http://fermicloud077.fnal.gov:9090
-start = 910
+start = 26052
 timeout = 1400
 debug = False
 summary = True
-maxstart = 86400
+maxstart = 21625672
```

Comment:
This looks like significant increase, does it determine time-range for getting the results if regular activity failed ? Asking for such time range could have heavy impact on the workload of the toolkits - I suggest we don't go beyond 48 hours. If we need to fill a large missing time window, we might need to introduce some interleave between queries - maybe startup another collector instance that would perform "regular" collection over fixed time window.